### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by git-0r**
# Created ecom cart screen
ecom5

### CODE REVIEW

File: pages/cart_page.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                        alt="">
+                    <div class="card-data">
+                        <p class="card-title text-xl">The Brown Shoes</p>
+                        <p class="text-sm text-clr-gray">by Some Premium Brand</p>
+                        <p class="text-xl">&#8377; 1999 <span class="text-clr-gray  product-discount">50% OFF</span></p>
+                        <div class="d-flex">
+                            <p>Quantity: </p>
+                            <div class="d-flex">
+                                <p class="text-xl product-qty-control">-</p>
+                                <input class="product-qty-input" type="number">
```
Comment (Naman-Saxena1) [inline pages/cart_page.html]: Can Add min property, values are going negative

File: pages/cart_page.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                                <p class="text-xl product-qty-control">+</p>
```
Comment (Naman-Saxena1) [inline pages/cart_page.html]: It will be better if you use button rather than para.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: pages/cart_page.html (hunk 2 above)
Comment (git-0r) [reply pages/cart_page.html]: done

File: css/cart_page.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
+body {
+  max-width: 1440px;
```
Comment (Sahil25061999) [inline css/cart_page.css:2]: use rem

File: css/cart_page.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+.product-list .product-discount {
+  text-decoration: line-through;
+}
+
+.product-qty-input {
+  width: 3rem;
+  height: 1.5rem;
+  border-radius: var(--border-radius-sm);
+  margin: 0 0.5rem;
```
Comment (Sahil25061999) [inline css/cart_page.css:60]: use px if value is less than 10

File: css/cart_page.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  border: 1px solid var(--clr-black-mid);
+}
+
+.payment-info .order-btn {
+  width: 100%;
+}
+
+.banner {
+  width: 100%;
+  height: 400px;
```
Comment (Sahil25061999) [inline css/cart_page.css:89]: use rem here too.

File: css/cart_page.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+  background-color: var(--clr-white-disabled);
+}
+
+footer {
+  padding: 1rem;
+  border-radius: var(--border-radius-sm);
+  background-color: var(--clr-primary-outline);
+}
+
+@media screen and (max-width: 600px) {
```
Comment (Sahil25061999) [inline css/cart_page.css:117]: use em for media queries.

File: pages/cart_page.html [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+                        alt="">
+                    <div class="card-data">
+                        <p class="card-title text-xl">The Brown Shoes</p>
+                        <p class="text-sm text-clr-gray">by Some Premium Brand</p>
+                        <p class="text-xl">&#8377; 1999 <span class="text-clr-gray  product-discount">50% OFF</span></p>
+                        <div class="d-flex flex-justify-between flex-align-center">
+                            <p class="qty-tag">Quantity: </p>
+                            <div class="d-flex flex-center">
+                                <button class="text-xl product-qty-control">-</button>
+                                <input class="product-qty-input" type="number" min="1">
```
Comment (Sahil25061999) [inline pages/cart_page.html:65]: just a suggestion here to give a default value of 1.

**Review by Sahil25061999 — APPROVED** [review]
Can be merged with some minor changes, nice touch by adding responsiveness to the page.

**Review by Naman-Saxena1 — APPROVED** [review]
Looks good to merge 👍

**Pull request merged**